packages/cadl-ranch-specs/http/type/dictionary/mockapi.ts

```diff
@@ -23,6 +23,12 @@ function createServerTests(uri: string, data: any) {
       response: {
         status: 204,
       },
+      handler: (req) => {
+        req.expect.coercedBodyEquals(data);
```

Comment:
all the remainings are due to dotnet uses different but compatible datetime format. So, we need to use `coercedBodyEquals` instaed of exact match.